parlai/agents/transformer/modules.py

```diff
@@ -999,12 +1063,48 @@ def prepare_head(tensor):
             # key and value are the same, but query differs
             # self attention
             value = key
-        _, key_len, dim = key.size()
```

Comment:
Do you mind doing _key_len to avoid the lint but leave the semantics provided by thus

Reply:
Okay - I've changed this, but I've also changed `key_len` (later in the function) to `full_key_len` to distinguish it from `_key_len`